### PR TITLE
Update license header

### PR DESCRIPTION
**Pull request by slide**

### CODE REVIEW

File: Src/IronPythonTest/AssemblyInfo.cs [hunk 1]
```diff
@@ -24,7 +14,7 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
 [assembly: AssemblyProduct("IronPythonTest")]
-[assembly: AssemblyCopyright("� Microsoft Corporation.  All rights reserved.")]
+[assembly: AssemblyCopyright("� Microsoft Corporation.  All rights reserved.")]
```
Comment (slozier) [inline Src/IronPythonTest/AssemblyInfo.cs]: Copyright symbol is broken. Should probably read IronPython Contributors anyway...

Comment (slide) [reply Src/IronPythonTest/AssemblyInfo.cs]: Hmmm, it looks fine here, what is broken about it?

Comment (slozier) [reply Src/IronPythonTest/AssemblyInfo.cs]: Shows up as a diamond with a question mark over here.

**Comment by slide** [comment]
Is it better now?

**Pull request merged**